wordpress-mobile/WordPress-Android
# Login Rework: prologue screen visual touches

Finishes up the visual and textual touches for the login prologue screen.

* 2 loops per animation then auto-advance
* Using Material Design button style

![device](https://user-images.githubusercontent.com/1032913/27729305-0a523f1c-5d8e-11e7-8f81-944394d6b6e1.jpg)

To test:
* With the app data cleaned up, run the app (make sure it's a `wasabi` build variant)
* Observe the prologue screen. Notice the auto-advancing of the view-pager as well. All 5 animation screens should perform OK.
